### PR TITLE
feat(scl-transformation): automatic IEDName subscriber auto-complete

### PR DESCRIPTION
**Pull request by JakobVogelsang**

### CODE REVIEW

File: src/Importing.ts [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+        new: {
+          parent: doc!.querySelector(':root')!,
+          element: ied,
+          reference: doc!.querySelector(':root > DataTypeTemplates')!,
+        },
+      });
+
+      this.dispatchEvent(
+        newActionEvent({
+          title: 'Import IED ' + ied.getAttribute('name'),
```
Comment (c-dinkel) [inline src/Importing.ts:199]: There seems to be a translation missing here. Are there also translations missing in other places?

**Review by c-dinkel — CHANGES_REQUESTED** [review]

**Comment by c-dinkel** [comment]
This could use a rebase on the current main branch.

File: src/Importing.ts (hunk 1 above)
Comment (JakobVogelsang) [reply src/Importing.ts]: I found another one. Those were introduced with the importing mixin. Thanks for the hint!

File: test/integration/open-scd.test.ts [hunk 2]
```diff
@@ -251,7 +251,7 @@ describe('open-scd', () => {
         'No IED element in the file'
       );
     });
-    it('throws doblicate ied name error', async () => {
+    it('throws dublicate ied name error', async () => {
```
Comment (c-dinkel) [inline test/integration/open-scd.test.ts:254]: this is actually spelt "duplicate" 😉

**Review by c-dinkel — APPROVED** [review]
Only a test spelling mistake left.

**Comment by c-dinkel** [comment]
Closes #82 .

**Comment by c-dinkel** [comment]
Sorry for the accidental premature closing of this request.

**Pull request merged**